docs/07_how-to/03_join-expose.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: 'How to join tables and make the results available'
+sidebar_label: 'Join and expose tables'
+id: ht-join-expose
+keywords: [views, tables, data server, request server]
+tags:
+    - views
+    - tables
+    - data server
+    - request server
+---
+
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+## Joining tables
+Typically, you design the data for your application so that there are neat places to store all the different types of data. You could have separate tables for INSTRUMENTS, CURRENCIES and COUNTERPARTIES, for example. And then you might have more tables for ORDERS, TRADES and CLIENT_DETAILS.
```

Comment:
Should we try educate here and say it's talking about second normal form https://www.freecodecamp.org/news/database-normalization-1nf-2nf-3nf-table-examples/#heading-the-first-normal-form-1nf 
?

Reply:
as well as keeping the phrasing as is (as I see it throughout) . I realised we want to appeal to more than engineers but equally engineers might turn their nose up.